pages/guide/08-start.md

```diff
@@ -0,0 +1,448 @@
+
+[Sample default semantic model config](https://github.com/yahoo/yavin/blob/master/packages/webservice/app/src/main/resources/demo-configs/models/tables/DemoTables.hjson)
+
+### How do I connect to a new database?
+
+Create/reuse the Hjson file for defining your data connection, example: ```DemoConnection.hjson``` in path ```yavin/packages/webservice/app/src/main/resources/demo-configs/db/sql/``` ***(Note: Multiple data connection can exist in a single Hjson file)***.
+
+**Sample H2 Connection config:**
+
+```
+{
+  dbconfigs: [
+    {
+      name: DemoH2Connection
+      url: jdbc:h2:mem:DemoDB;DB_CLOSE_DELAY=-1;
+      driver: org.h2.Driver
+      user: guest
+      dialect: H2
+    }
+  ]
+}
+
+```
+
+**Sample Presto Connection Config :**  
+
+```
+{
+  dbconfigs: [
+    {
+      name: DemoPrestoDBConnection
+      url: presto://PrestoDemo:8080/hive/demo;
+      driver: com.facebook.presto.jdbc.PrestoDriver
+      user: MrUser
+      dialect: PrestoDB
+    }
+  ]
+}
+
+```
+
+More information can be obtained at : [https://elide.io/pages/guide/v5/04-analytics.html#data-source-configuration](https://elide.io/pages/guide/v5/04-analytics.html#data-source-configuration)
+
+### How do I add a CSV file to Yavin?
+
+Direct upload of CSV to Yavin is not supported for Analytic Queries. ***You can upload the CSV to a database as a table and then create a Hjson config and access it via Yavin.*** After uploading the CSV file to HDFS and creating a Hive table on it. In path : ```yavin/packages/webservice/app/src/main/resources/demo-configs/db/sql/```. Create/reuse the Hjson file to point to the CSV file using Hive, example: ```CSV_Data_connection.hjson```
+
+**Sample Hive Connection config:**
+```
+{
+  dbconfigs: [
+    {
+      name: CSV_Data_connection
+      url: jdbc:hive:mem:DemoDB;DB_CLOSE_DELAY=-1;
+      driver: org.hive.Driver
+      user: guest
+      dialect: Hive
+    }
+  ]
+}
+
+```
+
+**If the CSV file you have is intended to be use alone**, then a similar approach to the demo CSV data provided by Yavin can be followed. Put a CSV file in path ```yavin/packages/webservice/app/src/main/resources/```. And in path : ```yavin/packages/webservice/app/src/main/resources/demo-configs/db/sql/```. Create/reuse the Hjson file to point to the CSV file using H2. And in path ```yavin/packages/webservice/app/src/main/resources/demo-configs/models/tables/``` Create/reuse the Hjson file for your model configuration.
+
+[Sample DB connection config](https://github.com/yahoo/yavin/blob/master/packages/webservice/app/src/main/resources/demo-configs/db/sql/DemoConnection.hjson)
+
+[Sample default semantic model config](https://github.com/yahoo/yavin/blob/master/packages/webservice/app/src/main/resources/demo-configs/models/tables/DemoTables.hjson)
+
+**If the CSV file is not used alone** then it has to be joined with fact data then it has to reside in the same database.
```

Comment:
If the CSV file has to be joined with other tables then it has to be loaded in the same database as those tables.